client/features/navbar/Navbar.js

```diff
@@ -18,23 +18,29 @@ const Navbar = () => {
       <nav>
         {isLoggedIn ? (
```

Comment:
Do you think you could tweak the ternary in this component or add a conditional to have one that handles when someone isLoggedIn and isAdmin? That could be the first condition checked and if that condition is passed, the navbar should display something that links to the full users page. 

If the user is a guest or is not an Admin, the other two conditionals should run.